packages/core/src/modules/drpc/DRPCMessagesApi.ts

```diff
@@ -0,0 +1,173 @@
+import type { DRPCRequest, DRPCRequestObject, DRPCResponse, DRPCResponseObject } from './messages'
+import type { DRPCMessageRecord } from './repository/DRPCMessageRecord'
+import type { ConnectionRecord } from '../connections'
+
+import { AgentContext } from '../../agent'
+import { MessageHandlerRegistry } from '../../agent/MessageHandlerRegistry'
+import { MessageSender } from '../../agent/MessageSender'
+import { OutboundMessageContext } from '../../agent/models'
+import { injectable } from '../../plugins'
+import { isValidDRPCRequestObject } from '../../utils/messageType'
+import { uuid } from '../../utils/uuid'
+import { ConnectionService } from '../connections'
+
+import { DRPCMessageRole } from './DRPCMessageRole'
+import { DRPCMessageHandler } from './handlers'
+import { DRPCErrorCode, DRPCRequestMessage, DRPCResponseMessage } from './messages'
+import { DRPCMessageService } from './services'
+
+@injectable()
+export class DRPCMessagesApi {
```

Comment:
```suggestion
export class DrpcMessagesApi {
```

To follow conventions in Credo

Reply:
I think that even the "Messages" word can be skipped (e.g. DrpcApi, DrpcService, etc.)